src/consensus/aft/raft_types.h

```diff
@@ -138,11 +137,9 @@ namespace aft
   struct RaftHeader
   {
     RaftMsgType msg;
```

Comment:
Why is the `from_node` removed from here, and passed separately as an argument in so many places (this is why we're now doing `recv_append_entries(node_id, append_entries, ...)` rather than just `recv_append_entries(append_entries, ...)`)?

Reply:
The `NodeId` used to be included in all consensus message headers, which was integrity protected by each channel separately. With the previous scheme, this data was fairly small but the new `NodeId` scheme means that we would be integrity-protecting a much bigger object, for each target node. Since the peer's `NodeId` is already included in each channel and is already an input to the integrity verification (i.e. `from_node` already routes to the correct channel), I've removed it from all the consensus headers. I don't have benchmarks to back this up but at the price of a "small" refactoring (which ended up being bigger than I thought), we don't do this anymore.